### ISSUE TITLE
Size limit on uploaded files?

### ISSUE THREAD

**Issue by mjmcginty** [blackcity/Backload#78]
I can upload 145MB files without issues but when I upload a 150MB file, it doesn't work correctly.  When blueimp's progress indicator gets to 100% the upload doesn't complete, progress goes back to 0 and it appears to upload it again, in a loop.

Apparently it's affected to some degree by "Maximum allowed content lengths" in IIS "Request Filtering", the limit on my dev system was 30MB, but I set it way higher than than the limit I am hitting.

Is there some effective limit in Backload or IIS, and if so, is there any way around it?

TIA,
MM

**Comment by Auersberg** [comment]
No, there is no size limit in Backload.
Are you using file chunking to upload larger files?
Is it an image that fails to upload?
What error message is reported?
Do you get any message, which can also be in the Json returned?

**Comment by mjmcginty** [comment]
It was not using chunked transfer.  I enabled that, now it works, I tested with a file a little under 1GB.  

So I'm closing the issue and calling it a day, even though I'd like to understand a little more about what was taking place...  I typed the rest of this before trying chunked transfer, leaving it in for reference...

Thanks again!

The file is not an image.

When I trap in client script, fileuploaddone event, data.result is null.  data.progress is there and loaded == total. data.textStatus i s "success" no error is indicated.

Something else that's strange, I set breakpoints in server code, in the HandleRequest function of my controller, before and after handler.Execute(). When it works correctly, the breakpoint after .Execute traps into the debugger (as expected.)  When it doesn't work, it doesn't trap after .Execute.  handler.Execute is wrapped in a try block, and I also have a breakpoint set on the first line of the catch block, it doesn't trap either.

Thanks
